заголовок нового проекта

Aleksandr-dl opened the pull request:
1ыы

Aleksandr-dl committed:
новый проект

1ыы

Aleksandr-dl committed:
сделал вроде

Кроме нижнего серого окна. Проверь завтра) а то я на сегодня что то устал)

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,99 @@
+      <article >
+        Designed in Slovenia
+      </article>
+      <a class="poster-link pourple-link" href="#"> EXPLORE NOW </a>
```
Тут либо article поменять на другую обертку(p или div), либо обернуть это все в article. Это же слайдер, сейчас просто реализован 1 слайд

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,99 @@
+        Modern finds for the modern home. Curated goods by unique designers around the world from fashion to furniture for kids and adults.
+      </p>
+    </section>
+    <section class="about purchase">
```
если в название класса ставить пробел, то будет 2 класса. между слов используй "-"

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,99 @@
+      <h3 class="delivery-title" Delivery></h3>
+      <a class="delivery"></a>
+      <a class="Guarantee"></a>
+      <a class="Credit"></a>
```
в названиях классов не должно быть больших букв

SergeyPechinov reviewed (changes requested):
Есть повторные правки, не стал писать тоже самое, пройдись по коду внимательно, и нужно расставить классы везде (каждый элемент должен иметь класс)

Aleksandr-dl committed:
первая страница

Привет, проверь первую страницу.
Не понимаю как в футоре сделать вводную страницу и серч, и заодно проверь все остальное, ошибки вроде исправил

Aleksandr-dl committed:
добавил кнопку

добавил кнопку

Aleksandr-dl committed:
Update index.html

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,139 @@
+      <h3 class="delivery-title" Delivery></h3>
+      <a class="delivery"></a>
+      <a class="guarantee"></a>
+      <a class="credit"></a>
```
если у атрибута a нет href мы не можем на него попасть с помощью клавиатуры

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,139 @@
+      </ul>
+    </section>
+    <section class="design-Life-Kids">
+      <h3 class="design-Life-Kids"Design Life Kids></h3>
```
тут ошибка в строке

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,139 @@
+      <a class="pourple-link" href="#"> buy now </a>
+    </section>
+    <section class="contacts">
+     <h3 class="contacts"Contacts></h3>
```
тут ошибка в строке

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,139 @@
+        <form action="email"></form>
+        <input type="email" name="email" id="#">
+        <sub>  join</sub>
+        </form>
```
1. 2 закрывающихся тега формы
2. join нужно оборачивать не в sub, а в button type="submit".
 _(Тег sub определяет текст в виде нижнего индекса. Нижней индекс выглядит как символ под строкой. Подобный текст может быть использован в химических формулах: H2O.),_

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,139 @@
+    </section>
+    <section class="map">
+      <!-- Тут будет карта -->
+    </section>    
```
1. нужно разметить блок, а карту вставить картинкой
2. в section обязательно должен присутствовать заголовок h2-h6, если его нет на макете, можно его разметить но сделать невидимым, либо использовать другой тег, если это возможно

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,139 @@
+    </section>    
+    </main> 
+    <footer>
+      <nav>
```
не нужно оборачивать весь footer в nav, а обернуть только нужные места

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,139 @@
+        <li><a class="2018 Holiday Catalog" href="#"> 2018 Holiday Catalog </a></li>
+        <li> <a class="Shop By Designer" href="#"> Shop By Designer </a></li>
+       </ul>
+       <ul>
```
ul не может содержать что-то кроме li

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,139 @@
+    <footer>
+      <nav>
+       <ul>
+        <a class="footer-navigation" href="#"> Navigation </a>
```
ul не может содержать что-то кроме li

Aleksandr-dl committed:
зеленый свет

что то там с футером

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,143 @@
+      <h3 class="delivery-title"> Delivery</h3>
+      <a class="delivery" href="#" ></a>
+      <a class="guarantee" href="#" ></a>
+      <a class="credit" href="#" ></a>
```
они не ведут на другую страницу, а только меняют заголовок и текст ниже, поэтому не должны быть ссылками

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,143 @@
+      </ul>
+    </section>
+    <section class="design-Life-Kids">
+      <h3 class="design-Life-Kids"> Design Life Kids </h3>
```
в названиях классов не должно быть больших букв

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,143 @@
+      <p>Some days i feel like i've been working on dlk my entire life and there are other days that i am completely
+        shocked that four years have flown by. It's simply magical.</p>
+      <img class="map-image" src="карта.png" alt="map">
+    </section>
```
почему он не в main?

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,143 @@
+           </ul>
+           <button>
+            <img src="Company-Rating" alt="stars">
+           </button>
```
1. Не является элементом навигации
2. Вообще не правильно сделан(всего 5 звезд), каждая звезда должна быть кнопкой, и текста "Company Rating:" нет

> при нажатии на определенную звезду будет ставиться рейтинг компании

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,143 @@
+          <li><a class="2018 Holiday Catalog" href="#"> Privacy Policy</a></li>
+          <li><a class="Shop By Designer" href="#"> Search </a></li>
+         </ul> 
+         <a class="footer-navigation" href="#"> Copyright © 2018 Dlk • Design Life Kids </a>   
```
это не ссылка, это просто текст

SergeyPechinov commented on index.html:
```diff
@@ -0,0 +1,143 @@
+        <li class="pertner-item"> <a class="social-link" href="#">Pinterest</a></li>
+        <li class="pertner-item"> <a class="social-link" href="#">instagramm</a></li>          
+       </ul>       
+        <a class="footer-navigation" href="#"> Web Design — Direct Line Development </a>                 
```
1 и 2 колонка обернуты в секцию, а 3-я почему-то нет

Aleksandr-dl committed:
Update index.html

Aleksandr-dl commented:
я вроде все исправил, кроме оценки сайта, подскажи, а то что то не понял, пожалуйста

Aleksandr-dl committed:
вроде кнопки сделал, но по моему не так

кнопки

Aleksandr-dl committed:
оценка компании

5 кнопок

Aleksandr-dl committed:
немгого стилей на хедер

посмотри пожалуйста со стиоями на хедер

Aleksandr-dl committed:
хедер и партнеров сделал

зацени хедер и партнеров

Aleksandr-dl committed:
немного блока

Aleksandr-dl committed:
вот последняя

на всякий случай.

Aleksandr-dl committed:
посмотри немного

Aleksandr-dl committed:
на всякий случай.

Одним глазом посмотри)

Aleksandr-dl committed:
практически все, кроме футора

Aleksandr-dl committed:
все седалал с первой страницей, за исключением мелочей

Aleksandr-dl committed:
вторая страница

картинки еще в два ряда не выставил

Aleksandr-dl committed:
модалочка

Aleksandr-dl committed:
все, кроме одной страницы

Aleksandr-dl committed:
все страницы
